src/remote.sh

```diff
@@ -48,13 +48,21 @@ function cmd_remotely()
   # Set default values if not specified
   remote=${remote:-"localhost"}
   port=${port:-"22"}
-  user=${user:-"root"}
+  user=${user:-$(whoami)}
   flag=${flag:-"HIGHLIGHT_CMD"}
 
-  composed_cmd="ssh -p $port $user@$remote \"$command\""
+  if [[ -v configurations['ssh_configfile'] ]]; then
+    composed_cmd="ssh -F ${configurations['ssh_configfile']} ${configurations['hostname']}"
+  else
+    composed_cmd="ssh -p $port $user@$remote"
+  fi
+
   if [[ "$bash_code" == 1 ]]; then
-    composed_cmd="ssh -p $port $user@$remote '$command'"
+    composed_cmd="$composed_cmd sudo '$command'"
+  else
+    composed_cmd="$composed_cmd sudo \"$command\""
```

Comment:
Suppose I don't assign any value to `ssh_configfile`, then cmd_remotely will run all commands as sudo*, right?

Reply:
yes!